myft/main.scss

```diff
@@ -71,6 +71,53 @@ $_n-ui-myft-is-silent: nUiHas('myft') == false or $_n-ui-myft-applied == true;
 		}
 	}
 
+	.n-myft-ui--prefer-group {
+		@include oGridRespondTo($from: 'M') {
+			padding-left: $spacing-unit;
+		}
```

Comment:
It didn't throw an error; should it? I was thinking the ordering was something like:
1. `include`s with no code blocks
2. `include`s with code blocks
3. other stuff